### PR TITLE
Objectstorage support

### PR DESCRIPTION
**Pull request by xevz**
I haven't updated `go.mod` since object storage suport isn't yet merged and tagged in go-glesys, but this was developed towards glesys/glesys-go@b92d4e381ddd.

### CODE REVIEW

**Comment by norrland** [comment]
We'll have objectstorage support in a tagged release soon.
Currently the code is in `master` branch of glesys-go.

File: examples/objectstorage/example.tf [hunk 1]
```diff
@@ -0,0 +1,11 @@
+resource "glesys_objectstorage_instance" "objectstore_example_fbg" {
+  count = 1
+  datacenter = "dc-sto1"
+  description = "tf-test-fbg-${count.index}-2"
```
Comment (norrland) [inline examples/objectstorage/example.tf:4]: s/fbg/sto/

**Comment by norrland** [comment]
Would you mind recreating this pr by rebasing on the main branch instead of doing merges.
To keep the git tree nice and clean.

**Comment by xevz** [comment]
@norrland Do you prefer every commit intact or should I squash some/all of them?

**Review by norrland — APPROVED** [review]
Looks good.

**Pull request merged**